mem/mem/protocol.go

```diff
@@ -495,6 +509,11 @@ func (b ControlMsgBuilder) Build() *ControlMsg {
 	m.Dst = b.dst
 	m.TrafficBytes = controlMsgByteOverhead
 
+	m.Enable = b.Enable
+	m.Drain = b.Drain
+	m.Flush = b.Flush
+	m.Invalid = b.Invalid
```

Comment:
Shall we rename Invalid to Invalidate? The words Enable, Drain, and Flush are all verbs.